webapp/src/js/controllers/reports.js

```diff
@@ -185,9 +187,9 @@ angular
         });
     };
 
-    var removeSelected = function(id) {
-      ctrl.removeSelected(id);
-      var index = _.findIndex(ctrl.selected, function(s) {
+    var removeSelectedReport = function(id) {
+      ctrl.removeSelectedReport(id);
+      var index = _.findIndex(ctrl.selectedReports, function(s) {
```

Comment:
These simple functions seem like nice candidates for lambdas. `s => s._id === id`

Reply:
Absolutely, I've mainly been focusing on refactoring the redux-relevant code, partially in order to not break anything, but I can start looking out for these obvious equivalences.